chore: native rendering datasource information. (#158)
ShipEngine/connect-website: native-rendering/documents-designer/data-source.md

```code
$['shipment']['tracking_number']
```

> For each document you need to select DataSource property. Select a 'package' if you want to print label for each package, and 'shipment' if you want only one label for whole shipment.

Reviewer: ```suggestion
For each document you need to select DataSource property value:
* **package** -  prints a label for each package in the shipment;
* **shipment** - prints only one label for the whole shipment.
```
To consider, and check whether formats and looks nice. Bold values are consistent with listings on  DesignerWindow page

```code
$['shipment']['tracking_number']
```

For each document you need to select DataSource property value:

> **package** -  prints a label for each package in the shipment;
> **shipment** - prints only one label for the whole shipment.

Reviewer: ```suggestion
* **package** -  prints a label for each package in the shipment;
* **shipment** - prints only one label for the whole shipment.
```
You lost punctuation

#### Current package reference

> If you select 'package' as DataSources, you can also reference to the current *Package* using prefix `@`

Reviewer: ```suggestion
If you select **package** as DataSource, you can also refer to the current *Package* using prefix `@`
```
At least 'DataSources' typo to be fixed
